cypress/fixtures/stubbing/ScenarioSharing/workspaces.js

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 
-import { WORKSPACE_EXAMPLE } from '../default/workspaces';
+import { WORKSPACE_EXAMPLE, USERS_LIST } from '../default';
+import { ROLES } from '../../../commons/constants/generic/TestConstants';
 
 export const WORKSPACE_NO_USERS_ = {
   ...WORKSPACE_EXAMPLE,
   id: 'W-stbbdbwry0',
   key: 'DemoBreweryNoUsers',
   security: {
-    default: '',
+    default: ROLES.SCENARIO.VIEWER,
     accessControlList: [],
   },
 };
```

Comment:
The definition and export of `WORKSPACE_NO_USERS_` was a mistake in a previous commit, it can safely be deleted :+1: